test_roll_analyzer.py

```
@@ -0,0 +1,171 @@
+    # Decision
+    decision_emoji = "🎲" if analysis["decision"] == "roll" else "📖"
+    decision_text = analysis["decision"].upper()
+
+    print(f"\n{decision_emoji} Decision: {decision_text}")
+    print(f"   Reasoning: {analysis.get('reasoning', 'N/A')}")
+    print(f"   Confidence: {analysis.get('confidence', 'N/A')}")
+
+    # Roll details (if decision was roll)
+    if analysis["decision"] == "roll" and analysis.get("roll_details"):
+        details = analysis["roll_details"]
+        print(f"\n🎯 Roll Details:")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)